platform/macos/app/resources/hillshade.svg

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="16"
+   height="16"
+   viewBox="0 0 16 16"
+   id="svg4148"
+   version="1.1"
+   inkscape:version="0.92.2 5c3e80d, 2017-08-06"
+   sodipodi:docname="hillshade.svg"
+   inkscape:export-filename="/Users/mxn/Desktop/symbol.png"
```

Comment:
Not a big deal, but Inkscape sure does toss a lot of cruft into SVGs. There are a variety of SVG optimizers/minifiers out there, if you felt so inclined.

Reply:
This would be good tail work, since all the SVGs in the platform/macos/app/resources/ folder are in Inkscape SVG format.

Reply:
#11051